src/v/pandaproxy/schema_registry/store.h

```diff
@@ -705,6 +716,65 @@ class store {
         return outcome::success();
     }
 
+    void setup_metrics() {
+        namespace sm = ss::metrics;
+        auto setup_common = [this]<typename MetricDef>() {
+            const auto make_subject_count = [this](is_deleted deleted) {
+                return ss::metrics::make_gauge(
+                         "subject_count",
```

Comment:
IIRC this is an in-memory count, right? Can we name the metric to reflect that? Otherwise this'll immediately cause issues and tickets when it doesn't match the expected count

Reply:
Done